src/structures/GuildChannel.js

```diff
@@ -156,6 +156,15 @@ class GuildChannel extends Channel {
     return [...sorted.values()].indexOf(sorted.get(this.id));
   }
 
+  /**
+   * Checks if the channel is not viewable by users with only the @everyone role.
+   * @type {boolean} true if private, false otherwise.
+   */
+  get private() {
+    const { everyone } = this.guild.roles;
+    return !this.permissionsFor(everyone).has(Permissions.FLAGS.VIEW_CHANNEL);
```

Comment:
may aswell just use `this.guild.roles.everyone` here instead of making a variable for it